public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -175,7 +171,8 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
       title: 'Dashboard',
       meta: {},
       editable: true,
-      body: state.body ?? new SceneFlexLayout({ children: [] }),
+      $timeRange: state.$timeRange ?? new SceneTimeRange({}),
```

Comment:
I don't fully understand this timerange addition. Was this something we missed previously?

Reply:
there was a test failing because it did not set it, and the change tracker was involved in the test and called sceneToSaveModel which does expect the time range

Reply:
not sure why that test does not fail in main though, probably hidden because something else in the test was not working properly in main